app/components/AfterIcoSumup.tsx

```diff
@@ -19,13 +20,21 @@ export const AfterIcoSumup = (props: IAfterIcoSumupProps) => {
   return (
     <Row className={`${styles.duringIco}`} data-test-id="during-ico-phase">
       <Col sm={5} className={styles.incentive}>
-        <h1>ICBM has finished!</h1>
+        <h1>Our ICBM ended on 17 December 2017, 11:00 am UTC</h1>
```

Comment:
we can't have this date hardcoded. Use finish date from store.

Reply:
I thought its not imporant as ICO is in progress and this date won't be changed. But ofc that's more elegant approach.

Reply:
we have two different deployments PUBLIC and WHITELISTED. Thats why we cant have this date hardcoded.